package.json

```diff
@@ -50,7 +50,7 @@
     "electron-rebuild": "^1.5.7",
     "ember-ajax": "^3.0.0",
     "ember-browserify": "^1.1.12",
-    "ember-cli": "2.17.1",
+    "ember-cli": "^2.18.0",
```

Comment:
Just wondering if you did re-run blueprints, as they recommend https://github.com/ember-cli/ember-cli/releases ?

Reply:
It wasn't meant to be part of the commit!
I was learning about semver, and that escaped!
I will rollback that!